tests/near-bindgen/assembly/test.ts

```diff
@@ -1,31 +1,43 @@
 import * as main from "./main";
-import * as model from "./model";
-import { base64 } from "near-runtime-ts";
+// import * as model from "./model";
+import { base64, logging } from "near-runtime-ts";
 import { FooBar } from "./model";
 import { u128 } from "bignum";
 
 //@ts-ignore
-@external("env", "log")
-declare function log(str: string): void;
+// @external("env", "log_utf8")
```

Comment:
?

Reply:
Yeah that's just an artifact when I was trying to print.  I thought I could match the export, but then realized I should just import `logging` I'll clean this up.